### ISSUE TITLE
Lots of temporary files left open aftzer

### ISSUE THREAD

**Issue by madmetz** [phpmyadmin/phpmyadmin#14604]
### Server configuration

- Operating system:
- Web server:
- Database version:
- PHP version:
- phpMyAdmin version:

### Client configuration

- Browser:
- Operating system:

**Comment by lem9** [comment]
Please show us some sample names of temporary files left there.

**Comment by ibennetch** [comment]
I don't use XAMPP so I'm not sure what's in the `./phpmyadmin/tmp` folder, but it could be the cache folder. When phpMyAdmin is run, some aspects of the pages are generated once and stored in the cache to speed up subsequent access. Those aren't really "temporary" files in the traditional sense, even though we do refer to the folder as "TempDir."

Probably this is the expected behavior of the caching functionality and unlikely to change, although if you can show us the sample names Marc's asked about we could confirm that for you.

**Comment by madmetz** [comment]
@ibennetch 
It seems you are right with the cache files.
BTW Xampp is a server collection for Windows including all the servers you need as a base for a cms.
After browsing some databases with phpMyAdmin the number of (cache) files increases to about 40 or more.
.
_Look at one of the filenames on the right side..._
![phpmyadmin_tmp](https://user-images.githubusercontent.com/42972320/45169906-df3e1f00-b1fe-11e8-93b4-a385c57d5e8c.jpg)
.
To me it makes sense to keep these files as long as phpMyAdmin is running.
But wouldn't it be better to delete these files after termination of phpMyAdmin ?
I'm not sure if this cache is still up-to-date after not using for some time.

**Comment by lem9** [comment]
There is no "termination" of phpMyAdmin as it's not a long running program. It runs just during the time of servicing a request.

**Comment by madmetz** [comment]
> ... as it's not a long running program.

That's right, but when I have to make some changes to a database I start phpMyAdmin in my webbrowser and when I'm done I exit phpMyAdmin by the icon with the half open door on the left side.
May be 'termination' is not the correct word in this context but please be tolerant.
English is not my native language. I'm sooooo sorry for that. :-)
